content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -45,9 +50,15 @@ The following steps enable plugging in the certificates and key into Citadel:
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files by using [Helm](/docs/setup/kubernetes/helm-install/)
+1.  Redeploy Citadel, which reads the certificates and key from the secret-mount files by using Helm
```

Comment:
This implies Citadel uses Helm to read the files. I think what you want instead is:

Redeploy Citadel using Helm. Citadel reads certificates...

Reply:
Done. Thanks for pointing this out.